src/CommandingOfficers/Patch.java

```diff
@@ -137,48 +179,8 @@ public void revertChanges(Commander commander)
     {
       // This will be called when the Commander removes this COModifier. It will remove the damage
       // modifier we added as well, so we just need to turn off the the damage-to-income listener.
-      GameEventListener.unregisterEventListener(listener);
+      coCast.myIncomeRatio = 0;
     }
   }
 
-  /**
-   * This GameEventListener will inspect published BattleEvents. If the owning Commander took
-   * part in the battle, then any damage done to the opponent will give income to the owning
-   * Commander. The amount of income is a fraction of value of the damage that was done, thus
-   * damaging more expensive units will grant more income.
-   */
-  private static class DamageDealtToIncomeConverter extends GameEventListener
```

Comment:
Not sure I'm a fan of moving this inside `LootAbility`. His passive is unrelated to the active abilities, and it seems to break separation of concerns to toss that in there.
Admittedly, `LootAbility` is already registered for events so I can see why that seems convenient, but I'd still keep it separate.

Reply:
Hmm.
The issue I ran into was that we now require a `GameInstance` reference to un/register, and you only get that at registration time (without fiddling with a whole lot of function APIs)
I suppose I could just make Patch have two listeners he subscribes, one to hold each of these functions?

Reply:
That's probably what I would have done.